scripts/generate_changelog.rb

```diff
@@ -131,9 +136,11 @@ def generate
 
     if entries.each_value.all? { |type| type[:entries].empty? }
       warn "No release notes for #{latest}..main"
-      exit 0
```

Comment:
I'd leave this in, just because if there aren't any release notes we don't actually know what the next version will be. Other than that, lgtm! 👍

Reply:
Whoops, good catch. I removed that line for testing (since we just did a release and there were no changes 😄) and forgot to put it back.